### PR TITLE
Bootstrap 4 + Jekyll transition

### PR DESCRIPTION
**Pull request by Shifterovich**
### Description

#544 

### HTML Preview

http://ptio.airdns.org:44449

### CODE REVIEW

**Comment by Shifterovich** [comment]
@kewde @bakku @Atavic @Hillside502 @beerisgood @beardog108 Can we merge?

**Comment by Shifterovich** [comment]
# Some new features

- **Awesome responsivity**
    - It works great on mobile. Viewport allows users to zoom. There's no horizontal scrollbar.
- Looks much nicer, much more modern.
    - Again, it's very nice even on mobile.
- Scrollspy
    - Shows the user at what section he's currently at
-  **Jekyll**
    - All those panels with recommended software are now ~7 line long `{% include %}`s.
- 100% valid HTML
    - We will add either a CI or a bot that would check validity of all pull requests.

## Screenshots

ptio airdns org_44449_ 1
ptio airdns org_44449_
[on mobile](https://imgur.com/a/DAj8hnN) (tables have a horizontal scrollbar and links break to avoid creating a horizontal scrollbar on the page)

**Comment by beardog108** [comment]
Based on preview, looks good to me. Nice work.

**Comment by Vincevrp** [comment]
I think the long-strings class is not working correctly. broken.

**Comment by Shifterovich** [comment]
Thanks for pointing that out. I'll fix that.

File: _layouts/default.html [hunk 1]
```diff
@@ -116,6 +116,7 @@
             <div class="dropdown-menu" aria-labelledby="softwareDropdown">
                 <a class="dropdown-item" href="#clients">Email Clients</a>
                 <a class="dropdown-item" href="#messaging">Email Alternatives</a>
+                <a class="dropdown-item" href="#search">Search Engines</a>
```
Comment (Shifterovich) [inline _layouts/default.html]: I resolved conflicts by ignoring all changes in master. I re-added this change manually.

screenshot from 2018-10-27 20-11-59

**Comment by Shifterovich** [comment]
I'll merge this PR now, but I'll close #544 once we resolve the messy navbar issue.

**Comment by Vincevrp** [comment]
> I'll merge this PR now, but I'll close #544 once we resolve the messy navbar issue.

Seems like bootstrap is broken because of a missing map file.

**Comment by Shifterovich** [comment]
Of course, this caused the site to break. I'm getting no errors in console, though.

**Comment by Vincevrp** [comment]
> Of course, this caused the site to break. I'm getting no errors in console, though.

I'll re-add the map file, one moment.

**Comment by Shifterovich** [comment]
I appended all URLs to JS and CSS with `?v=4`. Now I understand why custom.css had that `?v=1` in the URL to it.

**Pull request merged**